src/components/AddNote/AddNote.tsx

```diff
@@ -23,8 +23,10 @@ export interface AddNoteProps {
     event: SyntheticEvent<HTMLInputElement>,
```

Comment:
Why do we need the changes to the AddNote component?

Reply:
Because of the visual design. I needed to separate the 'Add note' button from the form. This required changes to the `AddNote` component.